.github/workflows/coffee.yml

```diff
@@ -9,7 +9,7 @@ jobs:
   coffee-break:
     runs-on: ubuntu-latest
     container:
-      image: golang:1.19
+      image: quay.io/redhat-appstudio-qe/qe-tools:latest
```

Comment:
Wondering, how is the quay image created? I think it would be worthwhile to add it to the [release workflow](https://github.com/redhat-appstudio/qe-tools/pull/98), what do you think? Otherwise LGTM after resolving conflicts

Reply:
Oh, thanks, I didn't notice that. Can I take it or do you want to do it yourself? Is there a Jira associated with it?

Reply:
Yes. [KFLUXBUGS-1157](https://issues.redhat.com/browse/KFLUXBUGS-1157)